skiko/src/jvmMain/cpp/common/BackendRenderTarget.cc

```diff
@@ -25,7 +31,8 @@ extern "C" JNIEXPORT jlong JNICALL Java_org_jetbrains_skia_BackendRenderTargetKt
     GrMTLHandle texture = reinterpret_cast<GrMTLHandle>(static_cast<uintptr_t>(texturePtr));
     GrMtlTextureInfo fbInfo;
     fbInfo.fTexture.retain(texture);
-    GrBackendRenderTarget* instance = new GrBackendRenderTarget(width, height, fbInfo);
+    GrBackendRenderTarget obj = GrBackendRenderTargets::MakeMtl(width, height, fbInfo);
+    auto instance = new GrBackendRenderTarget(obj);
```

Comment:
Inconsistent `auto` usage. I'm ok to use it, but it's better to align the approach because now it uses both ways in similar situations

Reply:
fixed. Using explicit types in this PR now.